oaknational/Oak-Web-Application
# feat: remove wobbly border from curriculum year jump buttons and add fade effect

## Description

Music year: 2010

- Remove the wobbly border from the year group buttons
- Added a fade effect to horizontal overflow of the button group
- Fixed scroll to year section scrolling ending prematurely bug

## Issue(s)

Fixes #[1038](https://www.notion.so/oaknationalacademy/Remove-wobbly-borders-from-year-jump-buttons-13b26cc4e1b1809fbca8e6bda2c6a1a5?pvs=4)

## How to test

Test using mobile and:

1. Go to https://deploy-preview-3029--oak-web-application.netlify.thenational.academy
2. Navigate to Science > KS3 & 4 > Select an exam board
3. Click through the years to see if the jump to section functionality works correctly
4. Scroll year button group horizontally to see if the fade effect works correctly

## Screenshots

Video:

https://github.com/user-attachments/assets/7427fc40-78d4-470a-9aab-fd56d669a053

🪧 In implementing the fade effect on horizontal overflow I hope that it doesn't matter that the effect also applies to the scroll bar. I tested this on my iOS device and the scroll bar does not appear when scrolling left or right on it, hence the effect does not apply to the scroll bar – meaning this is just a moot issue when we're emulating mobile on Chrome.

How it used to look (delete if n/a):
<img width="449" alt="image" src="https://github.com/user-attachments/assets/9df577de-98b1-44c8-a13b-a8a6fabce6ba">

How it should now look:
<img width="484" alt="image" src="https://github.com/user-attachments/assets/19e23ec9-e7db-425a-97a4-e55bf8f2c619">

## Checklist
